packages/fxa-auth-server/lib/senders/emails/partials/verificationReminder/index.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { automatedEmailNoAction, button } from '../../partials';
+
+export const verificationReminder = ({
+  title,
+  description,
+  subDescription,
+}: Record<any, string>) => `
+   <mj-include path="./lib/senders/emails/css/global.css" type="css" css-inline="inline" />
+   <mj-section>
+     <mj-column>
+     <mj-text css-class="header-text"><span data-l10n-id="<%- templateName %>-title">${title}</span></mj-text>
```

Comment:
Heh, I was going to suggest reducing the margin-bottom on this to better match the original template, and then I realized it's using an already established class. Yay for consistency!

I think we could perhaps reduce the padding on the first paragraph though. I think it makes sense to have a larger spacing between the header and text as you have it, as well as space between the last paragraph and button as you have it. 

The current/old template is showing `11px` margin-bottom here on the "confirmed it" paragraph:
<img width="360" alt="image" src="https://user-images.githubusercontent.com/13018240/129985251-39b147fa-8afa-4d9f-b8ec-deab9f324563.png">

Then in the new template, it's `20px`, coming from `primary-text`:
<img width="388" alt="image" src="https://user-images.githubusercontent.com/13018240/129985322-231069f8-ad3e-4e20-bc28-ec9f9d6e2c76.png">

I think this should be `12px`. I don't think we should just slap `primary-text mb-3` on there though since it overrides the `mb` from `text-primary` and we'll probably need to use that class again.

 I'm kind of torn on what we should call it, maybe because I think `primary-text` isn't necessarily the best class name (I know we were just pulling from the old templates though). 

We _could_ create a class called `body-text` for now which can go on the first paragraph...

```
.body-text div {
  @extend .text-sm;
  @extend .mb-3;
  @extend .font-sans;
  text-align: center !important;
}
```
... though, then the class name "primary-text" is almost weirder. What makes it "primary"? Maybe later in a follow up we can rename these classes.

I'm OK with that for now, but there may be a better solution. I don't think we can do something like this:

```
.primary-text:last-of-type div {
  @extend .mb-5;
}

.primary-text div {
  @extend .text-sm;
  @extend .mb-3;
  @extend .font-sans;
  text-align: center !important;
}
```
...because of the weird DOM output MJML does. I originally thought we could just add `@extend .mt-5;` to the `primary-button` class and then set `text-primary` to `mb-3` and let collapsing margins work in our favor, but I don't think that works either because of the DOM output.

Reply:
Yeah I was actually working on refactoring the styles a bit in the `verify` template PR (still WIP) so I'll just move those changes in here.